### ISSUE TITLE
With Braille panel open MuseScore Crashes when selecting multimeasure rest in a part and revealing an instrument

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#22414]
### Issue type

Crash or freeze

### Bug description

Update: only happens with Braille panel open.
In a part with a multimeasure rest, select the rest then make an instrument visible (one that would require the multi measure rest to recalculate), then MuseScore crashes

### Steps to reproduce

1. In preferences, make braille panel visible
2. Make a score with three parts
3. Add notes (in a way that creates the situation that the multimeasure rest will need to recalculate)
4. Open parts
5. Select multimeasure rest
6. Reveal other instrument that requires the multimeasure rest to recalculate

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/69917893/5ee8d70c-3c02-422b-9359-c6077776a252

### MuseScore Version

4.2.1

### Regression

No

### Operating system

Windows 11

### Additional context

Regression of 4.1.1

**Comment by lyrra** [comment]
can't reproduce on linux

**Comment by zacjansheski** [comment]
@lyrra interesting, I reproduced on Ubuntu with latest 4.3

**Comment by lyrra** [comment]
Just tried on 8f7ce94, do see alot of logging, but no warning.
<details>

```
20:21:27.500 | DEBUG | main_thread     | UiActionsRegister::updateEnabledAll | currentCtx: UiCtxProjectFocused
20:21:27.504 | DEBUG | main_thread     | NotationBraille::doBraille |  selected Multimeasure rest; Duration: Measure; Voice: 1
20:21:27.506 | DEBUG | main_thread     | UiActionsRegister::updateEnabledAll | currentCtx: UiCtxProjectFocused
20:21:30.497 | WARN  | main_thread     | MasterScore::rebuildExcerptsMidiMapping | rebuildExcerptsMidiMapping: no part in master score is linked with Oboe
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(TimeSig): there is already a TimeSig at track 0 tick 0
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 0
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 1920
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 3840
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 5760
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 7680
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 9600
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 11520
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 13440
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 15360
20:21:30.498 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 17280
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 19200
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 21120
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 23040
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 24960
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 26880
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 28800
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 30720
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 32640
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 34560
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 36480
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 38400
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 40320
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 42240
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 44160
20:21:30.499 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 46080
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 48000
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 49920
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 51840
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 53760
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 55680
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 57600
20:21:30.500 | DEBUG | main_thread     | Segment::checkElement | add(Rest): there is already a Rest at track 0 tick 59520
20:21:30.516 | DEBUG | main_thread     | Score::endCmd   | Undo stack current macro child count: 173
20:21:30.518 | DEBUG | main_thread     | UiActionsRegister::updateEnabledAll | currentCtx: UiCtxProjectFocused
20:21:30.521 | DEBUG | main_thread     | NotationBraille::doBraille |  selected Multimeasure rest; Duration: Measure; Voice: 1
20:21:30.525 | DEBUG | main_thread     | Part::setProperty | Part::setProperty: unknown id 2
20:21:30.541 | DEBUG | main_thread     | Score::endCmd   | Undo stack current macro child count: 2
20:21:30.543 | DEBUG | main_thread     | UiActionsRegister::updateEnabledAll | currentCtx: UiCtxProjectFocused
20:21:30.546 | DEBUG | main_thread     | NotationBraille::doBraille |  selected Multimeasure rest; Duration: Measure; Voice: 1
```
</details>

**Comment by bkunda** [comment]
Cannot reproduce on macOS using the latest nightly. 
@zacjansheski can you please recheck to confirm OS validity?

**Comment by zacjansheski** [comment]
No longer reproducible in master
Fixed in 4.4
